packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -249,6 +257,8 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
     _animation = createAnimation()
       ..addStatusListener(_handleStatusChanged);
     assert(_animation != null, '$runtimeType.createAnimation() returned null.');
+    _performanceModeRequestHandle =
+      SchedulerBinding.instance.createPerformanceModeRequest(this, ui.DartPerformanceMode.latency);
```

Comment:
nit: indent by two spaces.

Reply:
It's already indented with 2 spaces, maybe i'm missing something?